client/src/utils/distance.js

```diff
@@ -9,7 +9,9 @@ export function findDistance(point1, point2) {
   const pointDistance = distance.default(from, to, options);
   return pointDistance;
 }
+/* W.8.23.20 Q: Why is this not exported? Does it not need to be? 
```

Comment:
`getCurrentLocation()` does not need to be exported.  It is used inside `inOutLocation()`.  Only `findDistance()` and `inOutLocation()` are exported and used in `utils/api.js`.  These are used to determine distance to the locations on the cards.